frontend/globals/d.ts

```diff
@@ -0,0 +1,6 @@
+declare global {
```

Comment:
Any reason this ts is called d.ts?
Just not exactly clear what this does (to my non frontend developer brain at least!)

Reply:
This basically tells TypeScript that these are global types that it shouldn't need to complain about when it sees them in files in this folder

Reply:
I get what the code is doing, I'm questioning the naming of this file XD

Reply:
I did a lot of rejiging in the TypeScript Home branch, it's called overwrites.d.ts there :)